core/storage/classroom/gae_models.py

```diff
@@ -32,6 +32,15 @@
 (base_models,) = models.Registry.import_models([models.Names.BASE_MODEL])
 
 
+class ImageDict(TypedDict, total=False):
```

Comment:
@AFZL210 Shouldn't this be defined in the domain layer, as it is for all our other domain objects?

Reply:
My bad. I added this as earlier we were storing whole json data. but now I have removed it.

Reply:
Done.